Running in wrapper mode for '274'

Running python job.

[name: "CPU-QUOTA"
int_val: 76
int_val: 21
int_val: 40
int_val: 11
int_val: 81
int_val: 27
int_val: 82
, name: "DISK"
int_val: 26
int_val: 29
int_val: 11
int_val: 27
int_val: 19
int_val: 40
int_val: 37
, name: "MEMORY"
int_val: 12
int_val: 16
int_val: 39
int_val: 13
int_val: 13
int_val: 16
int_val: 39
, name: "NET"
int_val: 32
int_val: 36
int_val: 8
int_val: 31
int_val: 5
int_val: 5
int_val: 7
]
The paramters are {u'CPU-QUOTA': array([76, 21, 40, 11, 81, 27, 82]), u'NET': array([32, 36,  8, 31,  5,  5,  7]), u'DISK': array([26, 29, 11, 27, 19, 40, 37]), u'MEMORY': array([12, 16, 39, 13, 13, 16, 39])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 52500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 184320.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 24326963.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 600000000.0
Recovering the memory utilization
Setting cpu quota to 190000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 163840.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 21810380.8
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 450000000.0
Recovering the memory utilization
Setting cpu quota to 100000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 40960.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 9227468.8
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 1462500000.0
Recovering the memory utilization
Setting cpu quota to 190000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 163840.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 21810380.8
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 450000000.0
Recovering the memory utilization
Setting cpu quota to 100000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 40960.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 9227468.8
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 1462500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f40384a8bd8>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.36 seconds.
